source/_patterns/02-organisms/teaser-list/teaser-list.twig

```diff
@@ -0,0 +1,58 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list']) }) -%}
+
+<div {%- include('atoms-attributes') -%}>
+
+  {%- with title|default({}) only -%}
+    {%- block title -%}
+
+      {% if text is defined %}
+
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list__title']) }) -%}
+        {%- include 'atoms-heading' with {level: level|default(4)} -%}
+
+      {% endif %}
+
+    {%- endblock title -%}
+  {%- endwith -%}
+
+  {%- with list only -%}
+    {%- block list -%}
+
+      {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list__list']) }) -%}
+
+      <ol {%- include('atoms-attributes') -%}>
+
+        {%- for item in items -%}
+
+          {%- with item only -%}
+            {%- block item -%}
+
+              {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list__item']) }) -%}
+
+              <li {%- include('atoms-attributes') -%}>
+
+                {%- with content only -%}
+                  {%- block content -%}
+
+                    {%- if href is defined -%}
+                      {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list__link']) }) -%}
```

Comment:
Not needed?

Reply:
> Not needed?

Will remove. There have been instances where we have set a class to indicate that there is a link.